reports/helmcharts/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+source set_env.sh
+python main.py
```

Comment:
You need to replace `python main.py` with your own script name, `helm-chart-versions.sh`